Makefile

```diff
@@ -411,7 +411,7 @@ microk8s: check-microk8s ## Build cilium-dev docker image and import to microk8s
 	$(QUIET)./contrib/scripts/microk8s-import.sh $(LOCAL_OPERATOR_IMAGE)
 
 kind: ## Create a kind cluster for Cilium development.
-	$(QUIET)./contrib/scripts/kind.sh
+	SED=$(SED) $(QUIET)./contrib/scripts/kind.sh
```

Comment:
The `$(QUIET)` part *must* come at the start of the line, this breaks `make kind` in quiet mode.